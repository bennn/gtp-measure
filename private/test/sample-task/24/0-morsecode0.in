0100
1000
1111
1011
