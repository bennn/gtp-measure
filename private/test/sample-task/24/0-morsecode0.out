("0100" ("cpu time: 529 real time: 530 gc time: 84" "cpu time: 520 real time: 520 gc time: 82" "cpu time: 647 real time: 693 gc time: 131" "cpu time: 727 real time: 849 gc time: 113" "cpu time: 806 real time: 974 gc time: 119" "cpu time: 678 real time: 807 gc time: 97" "cpu time: 715 real time: 987 gc time: 125" "cpu time: 713 real time: 862 gc time: 108"))
("1000" ("cpu time: 593 real time: 617 gc time: 79" "cpu time: 688 real time: 733 gc time: 79" "cpu time: 794 real time: 991 gc time: 100" "cpu time: 821 real time: 982 gc time: 86" "cpu time: 865 real time: 1320 gc time: 123" "cpu time: 850 real time: 1306 gc time: 115" "cpu time: 784 real time: 1105 gc time: 110" "cpu time: 724 real time: 889 gc time: 115"))
("1111" ("cpu time: 563 real time: 576 gc time: 89" "cpu time: 500 real time: 505 gc time: 83" "cpu time: 485 real time: 505 gc time: 87" "cpu time: 607 real time: 712 gc time: 101" "cpu time: 510 real time: 525 gc time: 87" "cpu time: 534 real time: 605 gc time: 90" "cpu time: 478 real time: 489 gc time: 84" "cpu time: 476 real time: 488 gc time: 83"))
("1011" ("cpu time: 528 real time: 541 gc time: 84" "cpu time: 503 real time: 511 gc time: 77" "cpu time: 660 real time: 742 gc time: 103" "cpu time: 516 real time: 547 gc time: 79" "cpu time: 530 real time: 549 gc time: 94" "cpu time: 494 real time: 499 gc time: 79" "cpu time: 510 real time: 561 gc time: 88" "cpu time: 503 real time: 519 gc time: 77"))
